color.d

```diff
@@ -1480,15 +1480,29 @@ struct Rectangle {
 		this(upperLeft.x, upperLeft.y, upperLeft.x + size.width, upperLeft.y + size.height);
 	}
 
-	///
-	@property Point upperLeft() {
-		return Point(left, top);
-	}
-
-	///
-	@property Point lowerRight() {
-		return Point(right, bottom);
-	}
```

Comment:
Why change the indent style?
This few lines now differ from the whole rest of the file.

Reply:
yeah but meh, I'll change that myself, I'll allow it. but ilma make a different comment